data-source/jsonnet/northern-ireland/census_individual.jsonnet

```diff
@@ -0,0 +1,220 @@
+local apprenticeship = import 'blocks/individual/qualifications/apprenticeship.jsonnet';
+local degree = import 'blocks/individual/qualifications/degree.jsonnet';
+local gcse = import 'blocks/individual/qualifications/gcse.jsonnet';
+local nvq_level = import 'blocks/individual/qualifications/nvq_level.jsonnet';
+local other_qualifications = import 'blocks/individual/qualifications/other_qualifications.jsonnet';
+local qualifications = import 'blocks/individual/qualifications/qualifications.jsonnet';
+
+// employment
+local business_name = import '../common/blocks/individual/employment/business_name.jsonnet';
+local employer_address_depot = import '../common/blocks/individual/employment/employer_address_depot.jsonnet';
+local employer_address_workplace = import '../common/blocks/individual/employment/employer_address_workplace.jsonnet';
+local employers_business = import '../common/blocks/individual/employment/employers_business.jsonnet';
+local employment_status = import '../common/blocks/individual/employment/employment_status.jsonnet';
+local employment_type = import '../common/blocks/individual/employment/employment_type.jsonnet';
+local ever_worked = import '../common/blocks/individual/employment/ever_worked.jsonnet';
+local hours_worked = import '../common/blocks/individual/employment/hours_worked.jsonnet';
+local job_availability = import '../common/blocks/individual/employment/job_availability.jsonnet';
+local job_description = import '../common/blocks/individual/employment/job_description.jsonnet';
+local job_pending = import '../common/blocks/individual/employment/job_pending.jsonnet';
+local job_title = import '../common/blocks/individual/employment/job_title.jsonnet';
+local jobseeker = import '../common/blocks/individual/employment/jobseeker.jsonnet';
+local main_employment_block = import '../common/blocks/individual/employment/main_employment_block.jsonnet';
+local main_job_type = import '../common/blocks/individual/employment/main_job_type.jsonnet';
+local supervise = import '../common/blocks/individual/employment/supervise.jsonnet';
+local work_travel = import '../common/blocks/individual/employment/work_travel.jsonnet';
+local armed_forces = import 'blocks/individual/employment/armed_forces.jsonnet';
+local employer_type_of_address = import 'blocks/individual/employment/employer_type_of_address.jsonnet';
+
+// comments
+local comments = import '../common/blocks/comments.json';
+
+// submission confirmation
+local confirmation = import '../common/blocks/confirmation.json';
+
+function(region_code) {
+  mime_type: 'application/json/ons/eq',
+  schema_version: '0.0.1',
+  data_version: '0.0.3',
+  survey_id: 'census',
+  title: '2017 Census Test',
+  description: 'Census England Individual Schema',
+  theme: 'census',
+  legal_basis: 'Voluntary',
+  eq_id: 'census',
+  form_type: 'individual_gb_nir',
+  navigation: {
+    visible: false,
+  },
+  metadata: [
+    {
+      name: 'user_id',
+      validator: 'string',
```

Comment:
These have been changed to `"type": "string"` as part of https://github.com/ONSdigital/eq-survey-runner/pull/2076

Reply:
Good spot. Fixed.